{"name": "Проверка входа зарегистрированного пользователя с неверным логином и паролем", "status": "passed", "steps": [{"name": "Отправляем POST запрос", "status": "passed", "parameters": [{"name": "endpoint", "value": "'/api/auth/login'"}, {"name": "data", "value": "{'email': 'Не верные данные', 'password': 'DZLGkudrZozz'}"}], "start": 1735277493659, "stop": 1735277493999}, {"name": "Проверяем статус код", "status": "passed", "parameters": [{"name": "code", "value": "401"}], "start": 1735277493999, "stop": 1735277493999}], "attachments": [{"name": "stdout", "source": "98a1af14-5b8a-4e3a-9d40-d37a974f76d5-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "key", "value": "'email'"}], "start": 1735277493659, "stop": 1735277493999, "uuid": "a69e2bc5-ff79-4a0a-82d9-95c8dddaee2d", "historyId": "64f0f640c7cc695d332447c28a27ba57", "testCaseId": "0d4b7aaf7852bf668446d30a4080d733", "fullName": "tests.test_login_user.TestLoginUser#test_login_user_validation_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "DESKTOP-SQI1DNR"}, {"name": "thread", "value": "235160-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}